src/components/Fcrackzip/Fcrackzip.tsx

```diff
@@ -135,118 +149,134 @@ const Fcrackzip = () => {
+                                label={
+                                    "Verbose Mode: Generates extended information of files inside zip. (eg. name, size, etc.)"
+                                }
+                                checked={checkedVerbose}
+                                onChange={(e) => setCheckedVerbose(e.currentTarget.checked)}
                             />
-                            <span style={{ margin: "0 10px" }}></span>
                             <Checkbox
-                                label="Use Numeric Character"
-                                typeof="number"
-                                checked={useCharsetNumeric}
-                                onChange={(e) => setCharsetNumeric(e.currentTarget.checked)}
-                                //disabled={charSetDisabled}
-                            />
-                        </div>
-                        <div style={{ display: "flex", alignItems: "center" }}>
-                            <TextInput
-                                label={"Min Length"}
-                                type="number"
-                                required
-                                {...form.getInputProps("minLength")}
+                                label={"Show cracked password."}
+                                checked={checkedUnzip}
+                                onChange={(e) => setCheckedUnzip(e.currentTarget.checked)}
                             />
-                            <span style={{ margin: "0 10px" }}></span>
-                            <TextInput
-                                label={"Max Length"}
-                                type="number"
-                                required
-                                {...form.getInputProps("maxLength")}
-                            />
-                        </div>
-                        <Checkbox
-                            label={
-                                "Verbose Mode: Generates extended information of files inside zip. (eg. name, size, etc.)"
-                            }
-                            checked={checkedVerbose}
-                            onChange={(e) => setCheckedVerbose(e.currentTarget.checked)}
-                        />
-                        <Checkbox
-                            label={"Show cracked password."}
-                            checked={checkedUnzip}
-                            onChange={(e) => setCheckedUnzip(e.currentTarget.checked)}
-                        />
-                    </>
-                )}
-                {SaveOutputToTextFile(output)}
-                <Button type={"submit"}>Start Cracking!</Button>
-
-                <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
-                <Button onClick={clearOutput}>Clear Output</Button>
-            </Stack>
-        </form>
+                        </>
+                    )}
+                    {SaveOutputToTextFile(output)}
```

Comment:
This should be SaveOutputToTextFile_v2
```suggestion
                  {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
```